locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsWinRmMachineLocation.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package brooklyn.location.jclouds;
+
+import brooklyn.location.basic.WinRmMachineLocation;
+
+public class JcloudsWinRmMachineLocation extends WinRmMachineLocation {
```

Comment:
I wonder if it would be useful to have a `JcloudsMachineLocation` interface, with methods like `getNode()` and `getTemplate()`. That would also extend the `HasSubnetHostname` interface, with `getSubnetIp()` etc.

Ignore for this PR.